src/library_html5.js

```diff
@@ -204,7 +204,21 @@ var LibraryJSEvents = {
     },
 
     fullscreenEnabled: function() {
-      return document.fullscreenEnabled || document.mozFullScreenEnabled || document.webkitFullscreenEnabled || document.msFullscreenEnabled;
+      return document.fullscreenEnabled
+#if MIN_FIREFOX_VERSION <= 63
```

Comment:
side note, I think these checks would be clearer as `MIN_* < X` than `MIN_* <= X - 1` where `X` is where the modern version was added. That version is what you find on caniuse etc., and more natural to think about, in my opinion at least.

Reply:
<img width="522" alt="Screen Shot 2019-12-11 at 9 36 16 AM" src="https://user-images.githubusercontent.com/225351/70600987-c7bbc600-1bf9-11ea-8351-21b48b271c55.png">

I did write it in the way that caniuse shows - in this case Firefox <= 63 having `.mozFullScreenEnabled`. But I suppose it depends on just one's perspective on which box one is looking at... :)